opensuse-kubectl-docker/Dockerfile

```diff
@@ -19,8 +17,11 @@ RUN true \
     curl \
     wget \
     tar \
-    helm \
     direnv \
+    aaa_base bash-completion docker-bash-completion systemd-bash-completion \
+    && wget -q https://kubernetes-helm.storage.googleapis.com/helm-v2.4.2-linux-amd64.tar.gz -O - | tar xz -C /usr/local/bin --strip-components=1 linux-amd64/helm \
```

Comment:
Since we're getting whatever is the latest `kubectl` anyway… might want to look at https://github.com/kubernetes/helm/blob/e61ee5c54e1d24e791f516cb61059fc910befd1b/scripts/get#L79

Prehasp something liek this would work:
```bash
helm_version=$(curl -sSI https://github.com/kubernetes/helm/releases/latest | awk -F/ '/^Location:/ { print $NF }')
wget -q "https://kubernetes-helm.storage.googleapis.com/helm-${helm_version}-linux-amd64.tar.gz" -O - …
```